src/server/chat/commands.js

```diff
@@ -54,6 +54,14 @@ const commands = function() {
 
 
 		// Developer commands
+		{
+			// Starts the race early, in case you gotta go fast but the countdown is being a bully
```

Comment:
To add to this, it may also be worth it to let the bot respond to some of the commands, e.g. `!level` without a parameter would post the name of the level in the chat, or a non-dev who tries `!lotsofbots` could get a message stating the command is not allowed.
But yeah, something for another PR!